### ISSUE TITLE
[feature request] mp_hegrenade_penetration should be renamed

### ISSUE THREAD

**Issue by rust2** [rehlds/ReGameDLL_CS#742]
as written in `game.cfg`:
```
// Disable grenade damage through walls
// 0 - disabled
// 1 - enabled
//
// Default value: "0"
mp_hegrenade_penetration 0
```
IMHO cVar name should be changed to `mp_hegrenade_penetration_disable` or something like that, and it is confusing me because when i see `mp_hegrenade_penetration 1` I think grenades will deal damage to me. But it is opposite

**Comment by ShadowsAdi** [comment]
That is pretty clear.
`Disable grenade damage through walls ?` 
```
0 - function disabled
1 - function enabled
```

**Comment by rust2** [comment]
but cvar name should be self explaining

**Comment by StevenKal** [comment]
Yes, CVar name is illogical. Feature in-game should be swapped if name is kept.

**Comment by Giferns** [comment]
bad idea to touch it as it will break plugins that use this cvar

**Comment by StevenKal** [comment]
Is really there plugins relying on it (like for a custom damage radius code)? In such case renaming the CVar will also break such plugins.
In overal, changing just its internal behavior (swap) will break more the gameplay behavior until the admins change the CVar to the other value.

**Comment by Nord1cWarr1or** [comment]
The inverted name is acknowledged in the thread, but renaming (or inverting) the cvar would break existing plugins and configs that use it — same compatibility concern as noted by Giferns. The `game.cfg` documentation describes the actual semantics correctly. Closing as not planned (backward compatibility).
